sphinx_immaterial/inline_icons.py

```diff
@@ -0,0 +1,68 @@
+"""This module adds the md-icon role. This is specific to this this theme because,
+primarily, the icons bundled with this theme are supported (plus any svg files in the
+user project's static directories)."""
+from pathlib import Path
+from typing import Tuple, List
+from docutils import nodes
+from sphinx.application import Sphinx
+from sphinx.builders import Builder
+from sphinx.writers.html5 import HTML5Translator
+
+
+def read_svg_into_span(builder: Builder, icon_name: str, classes: List[str]) -> str:
+    """Reads the SVG data into a HTML span element."""
+    svg = Path(__file__).parent / ".icons" / icon_name
+    if not svg.exists():
+        static_paths: str = getattr(builder.config, "html_static_path")
```

Comment:
I think this type annotation is wrong --- should be `List[str]`

Also could skip the extra `if not static_paths` check

Reply:
type hint is wrong, good catch.